src/spyglass/utils/nwb_helper_fn.py

```diff
@@ -69,6 +69,14 @@ def get_nwb_file(nwb_file_path):
                 from ..common.common_dandi import DandiPath
 
                 dandi_key = {"filename": os.path.basename(nwb_file_path)}
+                if not DandiPath() & dandi_key:
+                    # Check if non-copied raw file is in Dandi
+                    dandi_key = {
+                        "filename": os.path.basename(nwb_file_path).replace(
```

Comment:
Slight preference for committing to either pathlib or os.path. I think we are primarily using pathlib.